packages/api/src/qm/visitor.ts

```diff
@@ -305,8 +305,10 @@ export async function traverse(
         continue; //ignore the following steps
       } else {
         //success or skip
-        question.value = inputResult.result;
-        inputs[question.name] = question.value;
+        if (question.type != "func") {
```

Comment:
No need to set value for func node since there's no input for this kind of question